### DISCUSSION TITLE
Timezone different than system

### DISCUSSION THREAD

**Discussion by michaelmedellin** [tagyoureit/nodejs-poolController#549] [App and O/S Config Questions]
Running njsPC in a container via Podman. I'm setting the container's tz to `--tz=America/Chicago` however the controller is showing two hours behind CDT as Mountain Standard Time. 

I opened a shell in the container and ran `date` and confirmed the timezone is set to CDT which is correct. The controller is clearly picking up a change since it's showing MST (since it's not in UTC which would be the case if the containers tz was not set at runtime). 

Anywhere else in the pool controller config where TZ data/state may be stored?

**Comment by tagyoureit** [comment]
<img width="626" alt="2022-07-19_14-48-03" src="https://user-images.githubusercontent.com/1646840/179854500-c6c806e5-e7b4-49e0-a446-ec5b4a7d3560.png">

Have you set it here?

**Reply by michaelmedellin** [reply]
I tried setting there with no luck (from the docs it sounds like that configuration isn’t actually persisted to config — did I read that wrong?). 

Not sure what changed, but I ended up having to reload all the containers on a fresh Raspbian Lite 64-bit image. That seemed to work and pick up the correct system time zone.

**Reply by tagyoureit** [reply]
The timezone is stored/persisted in the `./data/poolConfig.json` data file.  But glad you were able to get this working.

**Reply by michaelmedellin** [reply]
Sincerely appreciate the help
